src/lib.rs

```diff
@@ -198,6 +200,27 @@ fn build_poi(row: Row) -> Option<Poi> {
         return None;
     }
 
+    let properties = build_poi_properties(&row, &name).unwrap_or(vec![]);
+
+    let weight_wikidata = properties
+        .iter()
+        .find(|p| &p.key == "wikidata")
+        .map_or(0., |_p| 100.0);
```

Comment:
I think we should define this type of weights  as constants in a single place.  
Maybe at the top of the file, for the moment ?